app/controllers/concerns/idv/verify_info_concern.rb

```diff
@@ -213,6 +213,7 @@ def async_state_done(current_async_state)
 
       if form_response.success?
         save_threatmetrix_status(form_response)
+        save_source_check_vendor(form_response)
```

Comment:
A/N: I feel like these two methods could be merged into an `handle_success_updates_for_idv_session` method, but leaving as is for now.